.ci/scripts/boot_simulator.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+set -e
+
+# The name here must match create_simulator.sh
+readonly DEVICE_NAME=Flutter-iPhone
+
+xcrun simctl boot "$DEVICE_NAME" || :
+echo -e ""
+xcrun simctl list
```

Comment:
Do we want to keep this echo?

Reply:
It's not needed, but I liked it - it puts a newline between the `boot` output and `list` output. Just gives it a little separation